### PR TITLE
feat: display a single file for scripts created

### PR DESCRIPTION
**Pull request by rihp**
image
image

### CODE REVIEW

File: apps/browser/src/components/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -21,24 +21,48 @@ export interface SidebarProps {
...
+            <File key={i} files={scriptsGrouped[name]} />
           ))}
         </div>
         <Upload className="Workspace" onUpload={uploadUserFiles}>
           <h3>
             <FontAwesomeIcon icon={faFolder} style={{ marginRight: "10px" }} /> WORKSPACE
           </h3>
-          {userFiles.map((file, i) => (
-            <File file={file} />
+          {Object.keys(userFilesGrouped).map((name, i) => (
```
Comment (nerfZael) [inline apps/browser/src/components/Sidebar/Sidebar.tsx:64]: I don't think we should be applying the same logic for workspace files

**Review by nerfZael — CHANGES_REQUESTED** [review]

File: apps/browser/src/components/Sidebar/Sidebar.tsx (hunk 1 above)
Comment (rihp) [reply apps/browser/src/components/Sidebar/Sidebar.tsx]: Updated!

**Comment by rihp** [comment]
image
Updated!

**Review by nerfZael — APPROVED** [review]
I'll merge this since it works, but I think the scripts "explorer" should not be a "file explorer"
We should have a separate view for this

**Pull request merged**